lib/train/platforms/detect/helpers/os_linux.rb

```diff
@@ -37,6 +37,7 @@ def parse_os_release_info(raw)
 
       raw.lines.each_with_object({}) do |line, memo|
         line.strip!
+        next if line.start_with?("#")
```

Comment:
This line here will trip the linter. Change the `"` to `'` and it will pass.  If the linter fails, the CI fails, and if the CI fails, we can't accept it.

I understand if you son't want to change a quote sign after so long- I'll flag this as an adoption candidate.

Reply:
I have updated the code... Took a while to remember what is was about, since I have been running my fork in the meantime...